python/ray/resource_spec.py

```diff
@@ -165,6 +166,10 @@ def resolve(self, is_head, node_ip_address=None):
             if gpu_ids is not None:
                 num_gpus = min(num_gpus, len(gpu_ids))
 
+        info_string = _get_gpu_info_string()
+        gpu_types = _constraints_from_gpu_info(info_string)
```

Comment:
Can we add a try / catch with error logging and setting to empty resources here?